osu.Game.Rulesets.Osu/Mods/OsuModTransform.cs

```diff
@@ -41,27 +82,61 @@ private void applyTransform(DrawableHitObject drawable, ArmedState state)
                     return;
 
                 default:
-                    var hitObject = (OsuHitObject)drawable.HitObject;
-
-                    float appearDistance = (float)(hitObject.TimePreempt - hitObject.TimeFadeIn) / 2;
+                    var h = (OsuHitObject)drawable.HitObject;
+                    float appearDistance = (float)(h.TimePreempt - h.TimeFadeIn) / 2;
 
                     Vector2 originalPosition = drawable.Position;
                     Vector2 appearOffset = new Vector2(MathF.Cos(theta), MathF.Sin(theta)) * appearDistance;
 
                     // the - 1 and + 1 prevents the hit objects to appear in the wrong position.
-                    double appearTime = hitObject.StartTime - hitObject.TimePreempt - 1;
-                    double moveDuration = hitObject.TimePreempt + 1;
+                    double appearTime = h.StartTime - h.TimePreempt - 1;
+                    double moveDuration = h.TimePreempt / 2 + 1;
```

Comment:
What's going on here? Why is this half of the original?

Reply:
That's because I changed it in the emit setting but upon actually playing the original I realized it has the same issue so I applied it to both. The issue is without making the movement not last the whole preempt it's nigh unreadable above like ar 9.3 making the mod a full memory mod for anything above 9.7 ish I'd say even for the best of players , halving it makes it a bit more lenient. Still far harder than the equivalent AR without it ( 9.8 feels like 10.3 ISH to me ) but not fully memory except maybe around the 10.5-11 region, and Imo it also looks better but that's the least important thing here, I added this for playability

Reply:
So to confirm, this PR is not just adding a new setting but completely changing the behaviour of this mod? This change should probably be split out into its own PR if so..

Reply:
understandable, should i remove it from the original side of the mod or both ? ( because radiate frankly needs it to be even mildly "playable" above ar 9 and so does the original transform but slightly less due to the lower distances covered) , i wouldnt say it completely changes the mod it just speeds up the moves , obviously your call to make just trying to give a full picture of why it was done